### ISSUE TITLE
dde-dock cannot be changed to 'efficient' mode

### ISSUE THREAD

**Issue by shayaknyc** [linuxdeepin/dde-dock#238]
Something bizarre....I'm running Deepin 15.10. Restarted comp recently and dde-dock and entire desktop is stuck in 'fashion' mode. I right-clicked on the dock, Mode>Efficient and nothing happened. 

I installed and ran dconf-editor, and drilled down to /com/deepin/dde/dock/display-mode. I then toggled off "use default value", clicked on "Custom value" and changed it to 'efficient' and clicked on the check box on the bottom right and nothing happens.

I then loaded a terminal window and typed:
$ sudo apt-get remove dde-dock
it removed dde and dde-dock. Then I typed:
$ sudo apt-get install dde-dock
and it installed dde and dde-dock. Once that was complete, I then typed:
$ ps aux|grep dde-dock
Found the PID and then typed:
$ sudo kill [PID]
dde-dock shutdown then restarted, tried right-clicking on the dock, Mode>Efficient Mode and NUTHIN.

Then I typed:
$ sudo find / -name '*dde*.conf' -type f
and it found 4 files with dde in the name as conf files in /etc/deepin and /usr/share/dde-session-ui. None of the .conf files had any entry in it related to fashion or efficient mode so I left them as-is.

I've restarted the comp numerous times, to no avail. ANY IDEAS?!

**Comment by ciciolina** [comment]
try to downgrade the package, i have a similar problem with the clock, i put in 12h and the system change it to 24h every time. Im sure if you downgrade dde-dock and block to no upgrade you are fine. Its a temporal fix until next update.

**Comment by shayaknyc** [comment]
Hmmm.....that's just weird, nobody else on the Telegram group is having this issue......and they're running the same OS and Version....I'll try, but what's causing this? I even tried 'tail'ing the logfile for anything when I attempt to make the change and NOTHING appears there other than the mouse action of right-click.....very bizarre.....

**Comment by shayaknyc** [comment]
Ok, so no easy way to downgrade....looks like I'm going to have to wait? This doesn't make sense....

**Comment by BLumia** [comment]
@electricface please take a look, thanks.

**Comment by shayaknyc** [comment]
Hi all, quick update. Someone in the Deeping telegram group suggested I try creating a new user account and see if the efficient mode works there. So, I created a new user account, switched to the new user, after logging in and the initial intro setup, I was able to successfully switch modes to Efficient. 

Sssoooo, not really sure what's going on in my primary user account, but I'd like to be able to revert. I can say that I *USED* to have conky and topbar and cairo dock installed, but no longer. Could there be a vestige from those that is preventing the dde-dock from behaving properly? How would I go about finding info on this....

Thank you!
